db/schema.rb

```diff
@@ -1452,7 +1452,7 @@
     t.datetime "updated_at", precision: 6, null: false
     t.bigint "user_id", null: false
     t.boolean "welcome_notifications", default: true, null: false
-    t.index ["user_id"], name: "index_users_notification_settings_on_user_id"
+    t.index ["user_id"], name: "index_users_notification_settings_on_user_id", unique: true
```

Comment:
Are these schema changes supposed to be part of this PR?

Reply:
I think so, but the PR is missing the actual migration file

Reply:
Note: [The DUS PR](https://github.com/forem/forem/pull/13390) will go in first, which will resolve this apparent change

Reply:
Hmmmm I think this is fine but am confused as to why it is fine... I guess because the DUS PR has these changes added, but I would think that PR being merged wouldn't show that these lines are newly added in git. 🤔